client/pluginmanager/drivermanager/manager.go

```diff
@@ -258,10 +266,15 @@ func (m *manager) loadReattachConfigs() error {
 
 	if s != nil {
 		for name, c := range s.ReattachConfigs {
+			if m.isDriverBlocked(name) {
```

Comment:
Hmm I wonder if we should just shutdown the driver and log

Reply:
We now attempt to reattach and kill the plugin here